.github/workflows/scanner-dev-vuln-update.yaml

```diff
@@ -39,7 +40,7 @@ jobs:
         make -C scanner bin/updater
         go clean -cache -modcache
         ./scanner/bin/updater -output-dir=dev
-        gsutil cp -r "dev" "gs://scanner-v4-test/vulnerability-bundles"
+        gsutil cp -r "dev" "gs://definitions.stackrox.io/vulnerability-bundles"
```

Comment:
Should we prefix V4 stuff in that bucket, like, in a sub-directory? It's shared with V2 and might be harder to navigate, or we might inadvertently step on V2's foot.

Reply:
We made the decision to keep using definitions.stackrox.io (not prefix or sub folder) in https://issues.redhat.com/browse/ROX-20459

So probably this what we will do here

Reply:
I was originally going to say it's fine as-is, but it may be best to keep them separated via a `v4/` directory. What do you think?

Reply:
Yeah, I understand the concern. Currently we have definitions.stackrox.io/mapping and definitions.stackrox.io/vulnerabilties

Technically they are separated unless we want all v4 to be under one same folder v4.

Reply:
After moving to the new DNS name, we are likely to not change this ever, so namespacing per version is highly recommended, and cheaper to do now.